packages/flame_bloc/pubspec.yaml

```diff
@@ -23,7 +23,7 @@ dev_dependencies:
   dartdoc: ^6.3.0
   flame_lint: ^1.1.2
   flame_test: ^1.16.1
-  flutter_lints: ^2.0.1
+  flutter_lints: ^3.0.2
```

Comment:
Why is this using both flutter_lints and flame _lints?

Reply:
I imagine it is because this has flutter code? flame_lints doesn't include any flutter-specific rules.
(not sure though, I am just bumping the version)

Reply:
it is actually never used, so let me remove